Makefile

```diff
@@ -969,30 +972,15 @@ shell: $(GOBUILDER)
 .PHONY: linuxkit
 linuxkit: $(LINUXKIT)
 
-ensure-builder:
-	@if ! docker --context default container inspect linuxkit-builder >/dev/null 2>&1; then \
-	    echo "Container linuxkit-builder does not exist, creating..."; \
-	    docker --context default container run -d --name linuxkit-builder \
-	        --privileged moby/buildkit:$(BUILD_KIT_VERSION) \
-	        --allow-insecure-entitlement network.host \
-	        --debug --addr unix:///run/buildkit/buildkitd.sock; \
-	else \
-	    current_image=$$(docker --context default container inspect linuxkit-builder | jq -r '.[].Config.Image'); \
-	    if [ "$$current_image" != "moby/buildkit:$(BUILD_KIT_VERSION)" ]; then \
-	        echo "Recreating container (expected moby/buildkit:$(BUILD_KIT_VERSION), found $$current_image)"; \
-	        docker --context default container rm -f linuxkit-builder; \
-	        docker --context default container run -d --name linuxkit-builder \
-	            --privileged moby/buildkit:$(BUILD_KIT_VERSION) \
-	            --allow-insecure-entitlement network.host \
-	            --debug --addr unix:///run/buildkit/buildkitd.sock; \
-	    else \
-	        echo "Container linuxkit-builder is up-to-date"; \
-	    fi; \
-	fi
-
 LINUXKIT_SOURCE=https://github.com/linuxkit/linuxkit
+PARALLEL_BUILD_LOCK:=$(shell mktemp -u $(BUILD_DIR)/eve-parallel-build-XXXXXX)
+
+$(PARALLEL_BUILD_LOCK): $(BUILD_DIR)
```

Comment:
I don't quite get what the parallel build lock file is there for. It looks we use it only when doing `docker image prune`? Why do we need it? Doesn't docker itself as a daemon manage parallel access?